### PR TITLE
Bwdb 11059

### PR DESCRIPTION
**Pull request by jhough07**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

Adding CSP API and UI guides. Updating some of Import guides

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

File: campaigns/csp/about.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+|:---------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------|
+| [Reseller and Brand Dashboard UI Guide](guides/bandwidth10dlcBrandCspUiGuide.md)      | How to provision, manage and view your brands using the Bandwidth Dashboard   | 
+| [Reseller and Brand API Guide](guides/bandwidth10dlcBrandCspApiGuide.md)      | How to provision, manage and view your brands using Bandwidth APIs   | 
+| [Campaign UI Guide](guides/bandwidth10dlcCampaignCspUiGuide.md)      | How to provision, manage and view your 10DLC campaigns using the Bandwidth Dashboard   |               
+| [Campaign API Guide](guides/bandwidth10dlcCampaignCspApiGuide.md)    | How to provision and manage your 10DLC campaigns using Bandwidth APIs            |                             
+| [FAQs](guides/campaignFaqs.md)                                         | 10DLC CSP Campaigns Frequently Asked Questions                                 |                             
+
+## Important Notes
+1. We only support for campaigns that have been approved by all carriers participating through TCR.
```
Comment (ajrice6713) [inline campaigns/csp/about.md]: typo maybe? 
We only provide support for..
or
We only support...

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 2]
```diff
@@ -0,0 +1,973 @@
...
+```http
+HTTP/1.1 400 Bad Request
+HTTP/1.1 403 Unauthorized
+HTTP/1.1 429 Too Many Requests
+```
+
+{% endextendmethod %}
+
+## Update brand
+_Note_: Non-editable fields will be ignored and will not be updated.
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md:608]: Should we document which fields are non-editable?

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 3]
```diff
@@ -0,0 +1,973 @@
...
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">PUT</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: Should be 

`<code class="put">PUT</code>`

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 4]
```diff
@@ -0,0 +1,973 @@
...
+```
+
+{% endextendmethod %}
+
+## Fetch campaign settings
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">GET</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: Should be
`<code class="get">GET</code>`

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 5]
```diff
@@ -0,0 +1,973 @@
...
+```
+
+{% endextendmethod %}
+
+## Update campaign settings
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">PUT</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: `<code class="put">PUT</code>`

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 6]
```diff
@@ -0,0 +1,973 @@
...
+```
+
+{% endextendmethod %}
+
+## Fetch brand
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">GET</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: `<code class="get">PUT</code>`

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 7]
```diff
@@ -0,0 +1,973 @@
...
+
+{% endextendmethod %}
+
+## Fetch detailed brand list
+This endpoint returns a paginated brand list with full brand details.
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">GET</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/details?type={type}&page={page}&size={size}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: `<code class="get">GET</code>`

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 8]
```diff
@@ -0,0 +1,973 @@
...
+
+{% endextendmethod %}
+
+## Fetch abbreviated brand list
+This endpoint returns a complete, un-paginated, brand list with limited detail.
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">GET</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/details?type={type}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md]: `<code class="get">GET</code>`

File: campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md [hunk 9]
```diff
@@ -0,0 +1,875 @@
...
+```
+
+{% endextendmethod %}
+
+## Update campaign
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">PUT</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/{campaignId}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md]: `<code class="put">PUT</code>`

File: campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md [hunk 10]
```diff
@@ -0,0 +1,875 @@
...
+```
+
+{% endextendmethod %}
+
+## Fetch campaign
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">GET</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/{campaignId}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md]: `<code class="get">GET</code>`

File: campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md [hunk 11]
```diff
@@ -0,0 +1,875 @@
...
+```
+
+{% endextendmethod %}
+
+## Fetch campaign list
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">GET</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns?page={page}&size={size}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md]: `<code class="get">GET</code>`

File: campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md [hunk 12]
```diff
@@ -0,0 +1,875 @@
...
+                </MnoStatus>
+            </MnoStatusList>
+            <SharingStatus>true</SharingStatus>
+        </Campaign>
+    </Campaigns>
+    <TotalCount>17</TotalCount>
+</CampaignsResponse>
+```
+
+### Error Response
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md:558]: Is this response different if the CampaignManagement feature is enabled on the account but the user does not have the appropriate role?

File: campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md [hunk 13]
```diff
@@ -0,0 +1,875 @@
...
+
+{% endextendmethod %}
+
+## Fetch campaign list by brand
+This endpoint fetches a paginated list of all campaigns for a given brandId.
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">GET</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/campaigns?page=0&size=2`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md]: ``<code class="get">GET</code>

File: campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md [hunk 14]
```diff
@@ -0,0 +1,875 @@
...
+
+{% endextendmethod %}
+
+## Deactivate campaign
+This endpoint will update the campaign status to 'EXPIRED'.
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">DELETE</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/{campaignId}`
```
Comment (ajrice6713) [inline campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md]: `<code class="delete">DELETE</code>`

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md (hunk 2 above)
Comment (jhough07) [reply campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md:608]: Yeah that's a good call

File: campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md (hunk 12 above)
Comment (jhough07) [reply campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md:558]: Yeah will just have different error message

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Review by ajrice6713 — DISMISSED** [review]

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Review by ajrice6713 — DISMISSED** [review]

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11059.s3-website-us-east-1.amazonaws.com/

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**